GitCommands/GitCommands.csproj

```diff
@@ -35,6 +35,7 @@
     <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
     <TargetFrameworkProfile>
     </TargetFrameworkProfile>
+    <LangVersion>7.2</LangVersion>
```

Comment:
Since we are using the latest version, should we set the version here?

Reply:
try `latest`

Reply:
We can set this to `latest` to always be on the latest version.

But if @gerhardol's question was whether this attribute needs to be present at all, the answer is yes. By default, with a VS2017 project you compile using C# 7.0. The point releases are made optional, to reduce the obligation to test on newer compilers, which allows more regular compiler releases.

Differences between 7.2 and 7.0 are minor, but they do improve some aspects of the language.

Reply:
I preferred being explicit about the version to reduce potential for problems where different developers are on different versions of the compiler.

Reply:
I'm not against it, but my concern is maintenance overhead associated with the explicit specification.

As a thought, is there something we can set for the Appveyor to target (like 7.2) and keep "latest" in projects?

Reply:
The benefit of being explicit in the source is that it applies to all environments equally, be it your PC, mine, or the CI server. If in future the project wants to support 7.3 features, a quick search/replace should update it. Developers wanting to add this to new projects would have to do so only if they need features newer than 7.0.

I believe you can pass the `-langversion` argument to `msbuild` if you want to make it explict on AppVeyor though.

Reply:
❗️ *Never use latest*

Latest means everyone has a different version and there is no record of what the acceptable one actually is. The option exists for compatibility purposes only, and the only place it should ever be seen is in the compiler test suite that verifies it works.